src/TextLogging/RosImplementation/include/BipedalLocomotion/TextLogging/RosLogger.h

```diff
@@ -0,0 +1,117 @@
+/**
+ * @file RosLogger.h
+ * @authors Giulio Romualdi
+ * @copyright 2021 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the BSD-3-Clause license.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_TEXT_LOGGING_ROS_LOGGER_H
+#define BIPEDAL_LOCOMOTION_TEXT_LOGGING_ROS_LOGGER_H
+
+#include <memory>
+#include <mutex>
+#include <string>
+
+#include <spdlog/common.h>
+#include <spdlog/sinks/base_sink.h>
+
+#include <BipedalLocomotion/TextLogging/Logger.h>
+
+#include <rclcpp/rclcpp.hpp>
```

Comment:
```suggestion
#include <rclcpp/logging.hpp>
```

Nitpick: using non-catch all headers can reduce compilation times (but there is the possibility that in the future the macro is moved from that header).

Reply:
Done in https://github.com/ami-iit/bipedal-locomotion-framework/pull/587/commits/b6d6f5d8a8093230f66573721bc39b2cadc4c9c7